Formula/molten-vk.rb

```diff
@@ -76,24 +69,17 @@ def install
     mv "External/SPIRV-Tools", "External/glslang/External/spirv-tools"
     mv "External/SPIRV-Headers", "External/glslang/External/spirv-tools/external/spirv-headers"
 
-    mkdir "External/glslang/External/spirv-tools/build" do
-      # Required due to files being generated during build.
-      system "cmake", "..", *std_cmake_args
-      system "make"
-    end
-
-    xcodebuild "-project", "ExternalDependencies.xcodeproj",
-               "-scheme", "ExternalDependencies-macOS",
-               "-derivedDataPath", "External/build",
-               "SYMROOT=External/build", "OBJROOT=External/build",
-               "build"
+    # Have fetchDependencies build requirments
+    # These won't be downloaded if we have them already in-place
+    system "./fetchDependencies", "--macos"
```

Comment:
That means it could silently start downloading unversioned things in future updates though

Reply:
Not exactly unless a new dependency is missing as it only downloads if the dependency is missing.

But I can give it another go getting the dependencies building without using this.

Reply:
You mean if they add a new dependency ignore the first part of the above comment.